src/services/public-holidays.service.ts

```diff
@@ -10,7 +10,7 @@ export const getListOfPublicHolidays = async (year: number, country: string): Pr
 
   try {
     const { data: publicHolidays } = await axios.get<PublicHoliday[]>(
-      `${PUBLIC_HOLIDAYS_API_URL}/PublicHolidays/${year}/${country}`,
+      `${ PUBLIC_HOLIDAYS_API_URL }/PublicHolidays/${ year }/${ country }`,
```

Comment:
I think it's not the best idea to add spaces inside string literals. Look though all your code, please.